Address testing feedback from Session 1 and 2

coderbyheart opened the pull request:
This changes the implementation to fetch the
list of groups the user is captain of.
While the backend prevents the creation of more
than one group this implementation provides
some robustness against future changes.

The actual problem is that the backend
does not populate the groupId in the session
cookie, therefore it is always undefined.

Addresses more feedback from https://www.notion.so/distributeaid/Sara-and-Nicole-Testing-22-12-21-e6e28813c06f4dd9a0f54b47168af6ca

coderbyheart committed:
fix: correctly mark onboarding as completed

This changes the implementation to fetch the
list of groups the user is captain of.
While the backend prevents the creation of more
than one group this implementation provides
some robustness against future changes.

The actual problem is that the backend
does not populate the groupId in the session
cookie, therefore it is always undefined.

ghost commented:
### CodeSee Review Map:
[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/distributeaid/shipment-tracker/495/0131fe10/3e5799be225f631a2e8d1985662ab0083628a83b.svg)](https://app.codesee.io/r/reviews?pr=495&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fshipment-tracker)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=495&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fshipment-tracker)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fshipment-tracker)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

coderbyheart committed:
fix(auth): continue with logout if server rejects it

There is nothing to do here, so continue
removing the local authorization information

coderbyheart committed:
fix: show Create Shipment button only for admins

coderbyheart committed:
fix: ensure list of groups is up to date

coderbyheart committed:
fix: show Create Group button for admins

coderbyheart committed:
fix: do not load shipment exports for non-admins

Regular users only get to see the shipment details.
This fixes the issue where the shipment list would
forever be loading for a regular user

coderbyheart committed:
fix: hide shipment edit button for non-admins

coderbyheart committed:
fix: use english Hey

coderbyheart committed:
fix: update contact email

coderbyheart committed:
fix: remove Serbia and Montenegro

coderbyheart commented:
@deammer feel free to apply corrections if you see issues. No hard feelings!
